### PR TITLE
CLDR-14735 Dashboard dragging

### PR DESCRIPTION
**Pull request by btangmu**
-New file cldrDrag.js with new function cldrDrag.enable

-Fix bug in cldrAccount.getTransferLink, use the optional chaining operator (?.)

-Fix formatting of cldrLoad.js (in an earlier commit it was unformatted)

-Comment on warning for await in cldrGui.ensureSession

CLDR-14735

- [x] This PR completes the ticket.

### CODE REVIEW

**Comment by macchiati** [comment]
Better if Steven reviews the javascript, unless he is unavailable. One question: if we get this into smoketest now and there are problems, will it be easy to revert (if other PRs go in after)?

File: tools/cldr-apps/js/src/esm/cldrGui.js [hunk 1]
```diff
@@ -79,6 +80,8 @@ async function ensureSession() {
     body: JSON.stringify({}), // no email/password
   });
   if (response.ok) {
+    // TODO: address VS Code warning "'await' has no effect on the type of this expression"
+    // for the second "await" in the next line
     const logintoken = await (await response).json();
```
Comment (srl295) [inline tools/cldr-apps/js/src/esm/cldrGui.js:85]: oops, yes the 2nd is redundant w /the await on line 74 and should just be deleted

File: tools/cldr-apps/js/src/esm/cldrLoad.js [hunk 2]
```diff
@@ -281,7 +281,9 @@ function verifyJson(json, subkey) {
   } else if (json.err_code) {
     var msg_fmt = cldrRetry.format(json, subkey);
     var loadingChunk = cldrDom.createChunk("", "p", "errCodeMsg");
-    loadingChunk.appendChild(cldrDom.createChunk("","i","glyphicon glyphicon-remove-sign fgred"));
+    loadingChunk.appendChild(
+      cldrDom.createChunk("", "i", "glyphicon glyphicon-remove-sign fgred")
+    );
```
Comment (srl295) [inline tools/cldr-apps/js/src/esm/cldrLoad.js:286]: from an earlier PR. I've also applied this in another PR

**Review by srl295 — APPROVED** [review]
Looks fine to me. 

can be backed out just by commenting out  the call to `cldrDrag.enable()`  and i don't think it should conflct (other than the usual minor merge conflicts)

File: tools/cldr-apps/js/src/esm/cldrLoad.js (hunk 2 above)
Comment (macchiati) [reply tools/cldr-apps/js/src/esm/cldrLoad.js:286]: ok, thanks!

**Comment by manuelco-cldr** [comment]
Dragging is not smooth, not precise - seems to have a mind of its own but it is good enough.

**Pull request merged**